CodePush.m

```diff
@@ -130,16 +117,17 @@ - (void)initializeUpdateAfterRestart
     
     if (pendingUpdate) {
         _isFirstRunAfterUpdate = YES;
-        int rollbackTimeout = [pendingUpdate[PendingUpdateRollbackTimeoutKey] intValue];
-        if (0 != rollbackTimeout) {
-            dispatch_async(dispatch_get_main_queue(), ^{
-                [self startRollbackTimer:rollbackTimeout];
-            });
+        BOOL wasInitialized = [pendingUpdate[PendingUpdateWasInitializedKey] boolValue];
+        if (wasInitialized) {
```

Comment:
Not crazy about this name.  I think a better name would be "isUpdateLoading."  From the point that the bridge reloads to now, that is what's happening.  If the value is true here, then the update loaded successfully.